examples/react/basic-eslint-plugin/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@tanstack/query-example-react-basic",
+  "version": "0.0.1",
+  "main": "src/index.jsx",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "axios": "^0.21.1",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
+    "@tanstack/react-query": "^4.7.1",
+    "@tanstack/react-query-devtools": "^4.7.1"
+  },
+  "devDependencies": {
+    "@tanstack/eslint-plugin-query": "^1.0.0",
```

Comment:
I don't think that we need a dedicated example for the lint plugin. would we actually see somewhere in the code that the lint rule shows a warning?

If you want, we can add the eslint package just to the already existing basic example?